docs/getting-started-kubernetes.rst

```diff
@@ -857,19 +857,87 @@ The ``pod.yaml`` configuration file for such a deployment:
 SR-IOV Network Operator Deployment – Parallel Node Configuration for SR-IOV
 ---------------------------------------------------------------------------
 
-.. warning:: This is a Tech Preview feature, which is supported only for Vanilla Kubernetes deployments with SR-IOV Network Operator.
+.. warning:: This is a feature is supported only for Vanilla Kubernetes deployments with SR-IOV Network Operator.
```

Comment:
Suggestion:
This feature is supported only for....

Reply:
done